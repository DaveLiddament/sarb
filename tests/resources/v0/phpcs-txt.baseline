{"historyAnalyser":"git","historyMarker":"0f078b7d445f661991605b91d8ddb9c7f796242d","resultsParser":"phpcodesniffer-full","analysisResults":[{"lineNumber":2,"fileName":"src\/Domain\/Common\/InvalidPathException.php","type":"Missing file doc comment","message":"Missing file doc comment","fullDetails":"{\"fullFileName\":\"\\\/vagrant\\\/src\\\/Domain\\\/Common\\\/InvalidPathException.php\",\"level\":\"ERROR\",\"fixable\":false}"},{"lineNumber":7,"fileName":"src\/Domain\/Common\/InvalidPathException.php","type":"Missing doc comment for class InvalidPathException","message":"Missing doc comment for class InvalidPathException","fullDetails":"{\"fullFileName\":\"\\\/vagrant\\\/src\\\/Domain\\\/Common\\\/InvalidPathException.php\",\"level\":\"ERROR\",\"fixable\":false}"},{"lineNumber":9,"fileName":"src\/Domain\/Common\/InvalidPathException.php","type":"Missing doc comment for function __construct()","message":"Missing doc comment for function __construct()","fullDetails":"{\"fullFileName\":\"\\\/vagrant\\\/src\\\/Domain\\\/Common\\\/InvalidPathException.php\",\"level\":\"ERROR\",\"fixable\":false}"},{"lineNumber":11,"fileName":"src\/Domain\/Common\/InvalidPathException.php","type":"Line exceeds 85 characters; contains 109 characters","message":"Line exceeds 85 characters; contains 109 characters","fullDetails":"{\"fullFileName\":\"\\\/vagrant\\\/src\\\/Domain\\\/Common\\\/InvalidPathException.php\",\"level\":\"WARNING\",\"fixable\":false}"},{"lineNumber":8,"fileName":"src\/Domain\/BaseLiner\/BaseLineImporter.php","type":"Line exceeds 85 characters; contains 113 characters","message":"Line exceeds 85 characters; contains 113 characters","fullDetails":"{\"fullFileName\":\"\\\/vagrant\\\/src\\\/Domain\\\/BaseLiner\\\/BaseLineImporter.php\",\"level\":\"WARNING\",\"fixable\":false}"},{"lineNumber":52,"fileName":"src\/Domain\/BaseLiner\/BaseLineImporter.php","type":"Expected 18 spaces after parameter type; 1 found","message":"Expected 18 spaces after parameter type; 1 found","fullDetails":"{\"fullFileName\":\"\\\/vagrant\\\/src\\\/Domain\\\/BaseLiner\\\/BaseLineImporter.php\",\"level\":\"ERROR\",\"fixable\":true}"}]}