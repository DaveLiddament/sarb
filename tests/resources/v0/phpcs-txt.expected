{
    "historyAnalyser": "git",
    "historyMarker": "0f078b7d445f661991605b91d8ddb9c7f796242d",
    "resultsParser": "phpcodesniffer-json",
    "analysisResults": [
        {
            "lineNumber": 2,
            "fileName": "src\/Domain\/Common\/InvalidPathException.php",
            "type": "Missing file doc comment",
            "message": "Missing file doc comment"
        },
        {
            "lineNumber": 7,
            "fileName": "src\/Domain\/Common\/InvalidPathException.php",
            "type": "Missing doc comment for class InvalidPathException",
            "message": "Missing doc comment for class InvalidPathException"
        },
        {
            "lineNumber": 9,
            "fileName": "src\/Domain\/Common\/InvalidPathException.php",
            "type": "Missing doc comment for function __construct()",
            "message": "Missing doc comment for function __construct()"
        },
        {
            "lineNumber": 11,
            "fileName": "src\/Domain\/Common\/InvalidPathException.php",
            "type": "Line exceeds 85 characters; contains 109 characters",
            "message": "Line exceeds 85 characters; contains 109 characters"
        },
        {
            "lineNumber": 8,
            "fileName": "src\/Domain\/BaseLiner\/BaseLineImporter.php",
            "type": "Line exceeds 85 characters; contains 113 characters",
            "message": "Line exceeds 85 characters; contains 113 characters"
        },
        {
            "lineNumber": 52,
            "fileName": "src\/Domain\/BaseLiner\/BaseLineImporter.php",
            "type": "Expected 18 spaces after parameter type; 1 found",
            "message": "Expected 18 spaces after parameter type; 1 found"
        }
    ]
}